hazelcast/src/main/java/com/hazelcast/config/CacheSimpleConfig.java

```diff
@@ -99,6 +99,9 @@
 
     private String mergePolicy = BuiltInCacheMergePolicies.getDefault().getImplementationClassName();
 
+    private boolean hotRestartEnabled;
+
+    //CHECKSTYLE:OFF
     public CacheSimpleConfig(CacheSimpleConfig cacheSimpleConfig) {
```

Comment:
Instead of the `CHECKSTYLE:OFF` sledgehammer I would prefer a focused `@SuppressWarnings("checkstyle:whatever")`.